pallets/parachain-staking/src/lib.rs

```diff
@@ -689,15 +698,29 @@ pub mod pallet {
 				parachain_bond_reserve_percent: Default::default(),
 				blocks_per_round: 1u32,
 				num_selected_candidates: T::MinSelectedCandidates::get(),
+				rewards_account: Default::default(),
 			}
 		}
 	}
 
 	#[pallet::genesis_build]
 	impl<T: Config> BuildGenesisConfig for GenesisConfig<T> {
 		fn build(&self) {
+			RewardsAccount::<T>::set(self.rewards_account.clone());
+
 			assert!(self.blocks_per_round > 0, "Blocks per round must be > 0");
-			<InflationConfig<T>>::put(self.inflation_config.clone());
+
+			// Set inflation configuration
+			let mut inflation_config = self.inflation_config.clone();
+			// if all the round values are 0, derive them from the annual values
+			if inflation_config.round.min.is_zero() &&
```

Comment:
can you explain again why this check is needed? do we allow round values that are non-zero as an input?

Reply:
it's the best way I found to communicate the user that those parameters are not under his control that avoid a refactoring

Reply:
but now it is possible to pass any non-zero value for round, so _user_ is in control? if you just always derive round values from annual you don't need to make any refactoring of the struct, no?